packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -160,7 +160,7 @@ const AuthDefault = ({
             key={provider}
             provider={provider}
             label={shouldLogin ? 'Log in with' : 'Sign up with'}
-            className="mb-1"
+            className="mb-1 border-0"
```

Comment:
Does this not overwrite the className in the providers map though? 👀

Reply:
Yes it does i was leaving this pages setup as it was, but i could remove this and deal with the spacing (margin) with gap or space-y which would be better?

Reply:
I mean more for this it won't take the border color then right? 
As we never merge the -1 border now.

Reply:
This doesn't get used on the google button as we are passing classNames in the providers map, so overrides this classNames to the one supplied in the map as its passed in { ...props } at the bottom of the props. I have confirmed it works currently as can be seen in the screenshots above :)

Reply:
I have refactored this now so will resubmit for review in a few